client/src/App.jsx

```diff
@@ -4,6 +4,8 @@ import Home from "./components/Home";
 import UserProfile from "./components/UserProfile";
 import Mentors from "./components/Mentors";
 import Contact from "./components/ContactPage";
+import SignUp from "./components/SignUpPage";
```

Comment:
Imports are a bit confusing here you import the component `Contact ` as `SignUp` from the file `SignUpPage`